### ISSUE TITLE
LayoutLM: 0.176 F1-score on the FUNSD

### ISSUE THREAD

**Issue by tengerye** [microsoft/unilm#298]
**Describe the bug**
Model I am using LayoutLM:

The problem arises when using:
* [ ] the official example scripts: (give details below)
* [x] my own modified scripts: (give details below)

**To Reproduce**
CUDA_VISIBLE_DEVICES=0 python run_seq_labeling.py --data_dir data --model_type layoutlm --model_name_or_path microsoft/layoutlm-base-uncased --do_lower_case --max_seq_length 512 --do_train --num_train_epochs 100.0 --logging_steps 10 --save_steps -1 --output_dir model-output-2 --labels data/labels.txt --per_gpu_train_batch_size 12 --per_gpu_eval_batch_size 12 --overwrite_output_dir --do_eval

**Expected behavior**
![image](https://user-images.githubusercontent.com/15181821/108671719-d56a1c80-751b-11eb-87cd-65566e7115a4.png)

- Platform: SMP Debian 4.14.81.bm.26
- Python version: Python 3.6.12
- PyTorch version (GPU?): 1.5.0+cu101

**Comment by tengerye** [comment]
It seems the pre-trained model you upload to huggingface is different from that provided in your link.
